src/surveys/components/SelectResponse.tsx

```diff
@@ -0,0 +1,85 @@
+import { VNode, h } from 'preact';
```

Comment:
@afolasope  Can you change the name of this component to ChoiceResponse? Select sounds more like a dropdown.

Reply:
Done